src/Core/src/Fonts/FontManager.iOS.cs

```diff
@@ -167,7 +167,10 @@ UIFont CreateFont(Font font)
 			UIFont ApplyScaling(Font font, UIFont uiFont)
 			{
 				if (font.AutoScalingEnabled)
+				{
+					System.Diagnostics.Debug.Assert(OperatingSystem.IsIOSVersionAtLeast(11), "UIFontMetrics.DefaultMetrics is supported on 'ios' 11.0 and later");
 					return UIFontMetrics.DefaultMetrics.GetScaledFont(uiFont);
+				}
```

Comment:
Can we add a check into the if block and just use the fallback?

Reply:
Yes we can, I was not sure if the `font.AutoScalingEnabled` guarantees that this would work